lib/src/main/java/com/thenewboston/bank/datasource/BankDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package com.thenewboston.bank.datasource
+
+import com.thenewboston.bank.model.Bank
+import com.thenewboston.common.http.NetworkClient
+import com.thenewboston.common.http.Outcome
+import com.thenewboston.common.http.makeApiCall
+import io.ktor.client.request.*
+import io.ktor.utils.io.errors.*
+import javax.inject.Inject
+
+class BankDataSource @Inject constructor(private val networkClient: NetworkClient) {
+
+    suspend fun fetchBanks() = makeApiCall(
+        call = { banks() },
+        errorMessage = "Failed to retrieve banks"
+    )
+
+    private suspend fun banks(): Outcome<Bank> {
```

Comment:
Reading this, I'd expect the method to return a single `Bank`. How about calling it something like `BankList` and the `Result` class could then be called `Bank`?

The Django server also seems to call it "Bank List": http://143.110.137.54/banks

Reply:
Sure! makes sense to rename it.